debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -241,6 +245,20 @@ public ValueConverter converter(Column column, Field fieldDefn) {
         }
     }
 
+    private ValueConverter createArrayConverter(Column column, Field fieldDefn) {
+        final String elementTypeName = column.typeName().substring(1);
+        final String elementColumnName = column.name() + "-element";
```

Comment:
I was initially concerned about that name, but it seems it's not going to be actually used anywhere. So it seems fine.